calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php

```diff
@@ -132,29 +132,32 @@ protected function compile()
 		// Overwrite the page meta data (see #2853, #4955 and #87)
 		$responseContext = System::getContainer()->get(ResponseContextAccessor::class)->getResponseContext();
 
-		if ($responseContext instanceof WebpageResponseContext)
+		if ($responseContext && $responseContext->has(HtmlHeadBag::class))
```

Comment:
I'd use an explicit check here (and at the related places):

```suggestion
		if (null !== $responseContext && $responseContext->has(HtmlHeadBag::class))
```

Reply:
We don't really do that in legacy code.